numpy/__init__.pyi

```diff
@@ -3537,6 +3537,7 @@ arcsinh: _UFunc_Nin1_Nout1[L['arcsinh'], L[8], None]
 arctan2: _UFunc_Nin2_Nout1[L['arctan2'], L[5], None]
 arctan: _UFunc_Nin1_Nout1[L['arctan'], L[8], None]
 arctanh: _UFunc_Nin1_Nout1[L['arctanh'], L[8], None]
+bit_count: _UFunc_Nin1_Nout1[L['bit_count'], L[5], None]
```

Comment:
```suggestion
bit_count: _UFunc_Nin1_Nout1[L['bit_count'], L[10], None]
```
Small update to `ufunc.ntypes`, now that both signed and unsigned ints are supported.

Reply:
Ohh right, I missed this, thanks.